### DISCUSSION TITLE
Softphone and hardphone

### DISCUSSION THREAD

**Discussion by mpogudintrade** [sipsorcery-org/sipsorcery#697] [Q&A]
Hello

We have a softphone based on sipsorcery lib and a hardphone. Could you answer some questions about their compatibility?

1) Is it possible to use both phones at the same time with the same credentials and call/answer only on the one that is more convenient at the time.

2) Is it possible to intercept information (call status, caller info, where call was forwarded to etc.) from a softphone  about a conversation that happened or is happening on a hardphone.

**Comment by sipsorcery** [comment]
Hi,

The answers to both your questions depend on the SIP Server/Provider you are using. You'll need to check your SIP Server docs or ask your SIP Provider.

Regards,
Aaron

**Reply by mpogudintrade** [reply]
Thank you for the answer
